kubernetes/observability/run-configuration.md

```diff
@@ -0,0 +1,107 @@
+## Generate telemetry from Llamastack and vLLM
+
+### vLLM
+
+#### metrics
+
+For vLLM, metrics are generated by default and are exposed at `vllm-endpoint:port/metrics`. For a list of metrics,
```

Comment:
@sallyom can we follow up on that one jira ticket to see if there is an open data hub vllm image we could use that includes your fix?

Reply:
this shouldn't pause the merging but something we maybe make a note of to patch later